ntfy/backends/pushover.py

```diff
@@ -17,6 +25,25 @@ def notify(title, message, user_key,
     if device:
         data['device'] = device
 
+    if sound:
+        data['sound'] = sound
+
+    if priority != 0 and priority <= 2 and priority >= -2:
```

Comment:
Could you add an `else:` clause? Probably with something like `raise ValueError('priority must be an integer from -2 to 2')